modules/server/server-status/src/main/java/com/enonic/xp/server/impl/status/StatusServlet.java

```diff
@@ -107,18 +101,17 @@ private void serialize( final HttpServletRequest req, final HttpServletResponse
         final OutputStream out = res.getOutputStream();
         final StatusContextImpl context = new StatusContextImpl( req, out );
         reporter.report( context );
-        out.close();
     }
 
     private void serializeJson( final HttpServletResponse res, final int status, final JsonNode json )
         throws IOException
     {
+        res.reset();
         res.setStatus( status );
         res.setContentType( MediaType.JSON_UTF_8.toString() );
 
-        final PrintWriter out = res.getWriter();
-        out.println( json.toString() );
-        out.close();
+        final OutputStream out = res.getOutputStream();
+        out.write( json.toString().getBytes( StandardCharsets.UTF_8 ) );
```

Comment:
It is possible to save one copy if you open a response Writer and write a resulting string directly. 
Note that Jetty uses encoding ContentType - so we will be fine.
To get really mad about performance we can use JsonWriter but it would complicate the code